pkg/console/install_panels.go

```diff
@@ -741,8 +741,9 @@ func addNetworkPanel(c *Console) error {
 		if hostName == "" {
 			return "must specify hostname", nil
 		}
-		if errs := validation.IsQualifiedName(hostName); len(errs) > 0 {
-			return fmt.Sprintf("%s is not a valid hostname", hostName), nil
+		// ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L242-L245
+		if errs := validation.IsDNS1123Subdomain(hostName); len(errs) > 0 {
+			return fmt.Sprintf("%s is not a valid hostname. Please follow lowercase RFC 1123 subdomain.", hostName), nil
```

Comment:
With this message, it may be difficult for users to fix an invalid hostname.
Kubernetes API shows:
`$ kubectl create ns ABC`
`The Namespace "ABC" is invalid: metadata.name: Invalid value: "ABC": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')`

Reply:
For current version. I think it's hard to show long message. How about adding a `TODO` here and let us fix it when we change a layout framework.

![Screen Shot 2021-09-27 at 1 25 54 PM](https://user-images.githubusercontent.com/4927361/134851316-36c49edd-0bf3-4653-8142-085035bf07bb.png)

Reply:
let's cut it before the last `, and` for now and improve it in the next version with TODO.

Reply:
Added it. Thank you.

Reply:
If it does not fit a long error message? Will it cause trouble if users input a long hostname? Seems like we are printing the hostName from the user input.

Reply:
@gitlawr  Thanks for the reminder. I updated error message.